tendrl/monitoring_integration/grafana/dashboard_utils.py

```diff
@@ -115,8 +117,9 @@ def get_alert(alert_id):
     config = maps.NamedDict(NS.config.data)
     if utils.port_open(config.grafana_port, config.grafana_host):
         resp = get(
-            "http://{0}/grafana/api/alerts/{1}".format(
+            "http://{}:{}/api/alerts/{}".format(
                 config["grafana_host"],
+                config["grafana_port"],
```

Comment:
Why not `config.grafana_host` and `config.grafana_port` here? Its a named dict so it can be used this way.

Reply:
It is possible, i will modify the code

Reply:
Ack. Please verify the changes for all possible scenarios.